src/components/myEvents/MyEvents.tsx

```diff
@@ -130,10 +134,10 @@ const EventRow = ({ event }: EventRowProps) => {
               )}
               {hostNames && (
                 <div className="flex items-center gap-2">
-                  <Crown size={16} />
+                  <Crown size={16} className={isHost ? "text-primary" : ""} />
```

Comment:
For conditional classnames we should be using the cn util. 

`className={cn(isHost && "text-primary")}`
`cn("w-72 overflow-hidden text-ellipsis text-nowrap", isHost && "text-primary font-bold")